### PR TITLE
feat(azure): fix container latest tag missing results

### PR DESCRIPTION
**Pull request by maxamel**
This PR fixes ContainerTag check for azure pipelines.
We add a test to verify this check is extended.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/secrets/sanity/secrets/true_positive.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
     "secret_key": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
     "my_new_password": "F322a45xxmwov9bpgRhyuByXj2nxz7khS6yXQmfSaQCmwbTF2jpfgC56az3a",
     "test_pass": "z2b7k2cQfzc+yjP2K8cjuQ8uoorHBpEvC+XWhU3Z5+IdrPQYwr991Lj73xfZ+RA2GzC0wTedDTvb1C2NX+3Gpw==",
-    "pg_pass": "sup1rstr0ngpass2ForTT"
+    "pg_pass": "sup1rstr0ngpass2ForTT",
+    "secret_key": "THISISASECRETKEY1234=="
```
Comment (gruebel) [inline tests/secrets/sanity/secrets/true_positive.json:7]: is this related to the PR? 😄

Comment (maxamel) [reply tests/secrets/sanity/secrets/true_positive.json]: oops thought I removed that one, guess I haven't

**Review by gruebel — APPROVED** [review]
🍻

**Review by arielkru — APPROVED** [review]
